code/datums/elements/update_icon_updates_onmob.dm

```diff
@@ -25,5 +25,3 @@
 			M.update_held_items()
 		else
 			M.update_clothing((target.slot_flags|update_flags))
-			if(update_body)
-				M.update_body()
```

Comment:
Update clothing also calls update body. I'm still doing some tests to see if this breaks non-human mobs somehow

Reply:
looks good 
![image](https://github.com/user-attachments/assets/82e9a2ab-0897-4a4d-9cdb-264d36620079)